### ISSUE TITLE
[IOS-XE] Parser for "show mka statistics" does not work

### ISSUE THREAD

**Issue by ChristopherJHart** [CiscoTestAutomation/genieparser#852]
Sample CLI output:
```
R2#show mka statistics           

MKA Global Statistics
=====================
MKA Session Totals
   Secured.................... 2
   Fallback Secured........... 0
   Reauthentication Attempts.. 0

   Deleted (Secured).......... 0
   Keepalive Timeouts......... 0

CA Statistics
   Pairwise CAKs Derived...... 0
   Pairwise CAK Rekeys........ 0
   Group CAKs Generated....... 0
   Group CAKs Received........ 0

SA Statistics
   SAKs Generated............. 1
   SAKs Rekeyed............... 0
   SAKs Received.............. 1
   SAK Responses Received..... 1

MKPDU Statistics
   MKPDUs Validated & Rx...... 501325
      "Distributed SAK"..... 1
      "Distributed CAK"..... 0
   MKPDUs Transmitted......... 543079
      "Distributed SAK"..... 1
      "Distributed CAK"..... 0

MKA Error Counter Totals
========================
Session Failures
   Bring-up Failures................ 0
   Reauthentication Failures........ 0
   Duplicate Auth-Mgr Handle........ 0

SAK Failures
   SAK Generation................... 0
   Hash Key Generation.............. 0
   SAK Encryption/Wrap.............. 0
   SAK Decryption/Unwrap............ 0
   SAK Cipher Mismatch.............. 0

CA Failures
   Group CAK Generation............. 0
   Group CAK Encryption/Wrap........ 0
   Group CAK Decryption/Unwrap...... 0
   Pairwise CAK Derivation.......... 0
   CKN Derivation................... 0
   ICK Derivation................... 0
   KEK Derivation................... 0
   Invalid Peer MACsec Capability... 0

MACsec Failures
   Rx SC Creation................... 0
   Tx SC Creation................... 0
   Rx SA Installation............... 0
   Tx SA Installation............... 0

```
Traceback:
```
2024-04-29T13:13:31: %SCRIPT-ERROR: Traceback (most recent call last):
2024-04-29T13:13:31: %SCRIPT-ERROR:   File "/Users/chart2/Documents/Python/genieparser/src/genie/libs/parser/utils/unittests.py", line 580, in test_golden
2024-04-29T13:13:31: %SCRIPT-ERROR:     parsed_output = obj.parse(**arguments)
2024-04-29T13:13:31: %SCRIPT-ERROR:                     ^^^^^^^^^^^^^^^^^^^^^^
2024-04-29T13:13:31: %SCRIPT-ERROR:   File "src/genie/metaparser/_metaparser.py", line 342, in genie.metaparser._metaparser.MetaParser.parse
2024-04-29T13:13:31: %SCRIPT-ERROR:   File "src/genie/metaparser/_metaparser.py", line 322, in genie.metaparser._metaparser.MetaParser.parse
2024-04-29T13:13:31: %SCRIPT-ERROR:   File "src/genie/metaparser/util/schemaengine.py", line 419, in genie.metaparser.util.schemaengine.Schema.validate
2024-04-29T13:13:31: %SCRIPT-ERROR: genie.metaparser.util.exceptions.SchemaMissingKeyError: Missing keys: [['mka-error-counters', 'mkpdu-failures']]
```

**Comment by SohanTirpude** [comment]
Hello @ChristopherJHart,

It looks like a bug. Kindly give me some time to analyse this issue.

Thank you.

**Comment by Neeraja1bala** [comment]
Hello @ChristopherJHart ,

Expected output will get like below,

```
{
    "mka-session-totals": {
        "secured": 2,
        "fallback-secured": 0,
        "reauthentication-attempts": 0,
        "deleted-secured": 0,
        "keepalive-timeouts": 0
    },
    "ca-statistics": {
        "pairwise-caks-derived": 0,
        "pairwaise-cak-rekeys": 0,
        "group-caks-generated": 0,
        "group-caks-received": 0
    },
    "sa-statistics": {
        "saks-generated": 1,
        "saks-rekeyed": 0,
        "saks-received": 1,
        "sak-responses-received": 1
    },
    "mkpdu-statistics": {
        "mkpdu-received": {
            "distributed-sak": 1,
            "distributed-cak": 0
        },
        "mkpdus-validated-received": 501325,
        "mkpdu-transmitted": {
            "distributed-sak": 1,
            "distributed-cak": 0
        },
        "mkpdus-transmitted": 543079
    },
    "mka-error-counters": {
        "session-failures": {
            "bringup-failures": 0,
            "reauthentication-failures": 0,
            "duplicate-auth-mgr-handle": 0
        },
        "sak-failures": {
            "sak-generation": 0,
            "hash-key-generation": 0,
            "sak-encryption-wrap": 0,
            "sak-decryption-unwrap": 0,
            "sak-cipher-mismatch": 0
        },
        "ca-failures": {
            "group-sak-generation": 0,
            "group-cak-encryption-wrap": 0,
            "group-cak-decryption-unwrap": 0,
            "pairwise-cak-derivation": 0,
            "ckn-derivation": 0,
            "ick-derivation": 0,
            "kek-derivation": 0,
            "invalid-peer-macsec-capability": 0
        },
        "macsec-failures": {
            "rx-sc-creation": 0,
            "tx-sc-creation": 0,
            "rx-sa-installation": 0,
            "tx-sa-installation": 0
        }
    }
}
```

**Comment by ChristopherJHart** [comment]
@Neeraja1bala That output looks correct to me - thank you!

**Comment by Neeraja1bala** [comment]
Hello @ChristopherJHart,

The PR got merged now and the fix will be available in the upcoming pyats 24.5 release.

Thank you.

**Comment by ChristopherJHart** [comment]
Closing out this issue, as it has been resolved.
